smart-contract-usage-in-javascript-sdk.md

```diff
@@ -0,0 +1,326 @@
+# TUTORIAL: Smart Contract usage with the Javascript SDK
+
+This tutorial explains how to use the æternity javascrip SDK to create smart contracts and interact with them in the æternity network.
+
+## Prerequisites
+
+ - up to date npm and nodejs installed
+ - private and public key for an account with some balance ([create an account](account-creation-in-ae-cli.md); [get testnet tokens](get-testnet-tokens.md))
+
+## Setup
+
+1. Create a directory for the project and initialize it with npm:
+
+```
+mkdir aepp-smart-contract-usage-tutorial
+cd aepp-smart-contract-usage-tutorial
+npm init -y
+```
+
+2. Install the aepp-sdk dependency `npm install --save @aeternity/aepp-sdk`
+
+3. Create a example project file `index.js` in this directory
```

Comment:
Id change this from to an "Create an example entrypoint file `index.js` in this directory. We will use this file later to run our code."